src/widgets/icons.js

```diff
@@ -2,16 +2,17 @@
  * mSupply Mobile
  * Sustainable Solutions (NZ) Ltd. 2019
  */
+
 import React from 'react';
 
 import IonIcon from 'react-native-vector-icons/Ionicons';
 import FAIcon from 'react-native-vector-icons/FontAwesome';
 
 import { dataTableColors } from '../globalStyles';
 
-export const SortAscIcon = <FAIcon name="sort-asc" size={15} color="black" />;
-export const SortNeutralIcon = <FAIcon name="sort" size={15} colour="black" />;
-export const SortDescIcon = <FAIcon name="sort-desc" size={15} colour="black" />;
+export const SortAscIcon = <FAIcon name="sort-asc" size={15} style={{ marginRight: 10 }} />;
+export const SortNeutralIcon = <FAIcon name="sort" size={15} style={{ marginRight: 10 }} />;
+export const SortDescIcon = <FAIcon name="sort-desc" size={15} style={{ marginRight: 10 }} />;
```

Comment:
Margin left? Or does the title have margin right too?

Reply:
There are many ways to do this and IDK what's the best.

Reply:
argh this was a gooden. Took me awhile to figure out how `generic..` was getting the right margin on the icons hehe. I was really just taking the styles and applying them, trying to 'tidy' up where I could. 

There's certainly a lot of redrawing, but maybe one area of performance enhancing at a time? 🤣 

![image](https://user-images.githubusercontent.com/35858975/63400744-064e2a80-c429-11e9-86da-ecc1f408fdab.png)